tests/20-basic-component-check.py

```diff
@@ -22,34 +21,16 @@ class IntegrationTest(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        # Get the relative bundle path from the environment variable.
-        cls.bundle = os.getenv('BUNDLE', 'local.yaml')
-        # Create a path to the bundle based on this file's location.
-        cls.bundle_path = os.path.join(cls.bundle)
-        # Normalize the path to the bundle.
-        cls.bundle_path = os.path.abspath(cls.bundle_path)
-
-        print('Deploying bundle: {0}'.format(cls.bundle_path))
         cls.deployment = amulet.Deployment()
```

Comment:
This looks like an oversight. I don't see this code adding anything to the deployment. Am I missing something?